controllers/ratelimitpolicy_controller_test.go

```diff
@@ -53,11 +53,13 @@ var _ = Describe("RateLimitPolicy controller", func() {
 					Kind:  "HTTPRoute",
 					Name:  gatewayapiv1.ObjectName(routeName),
 				},
-				Limits: map[string]kuadrantv1beta2.Limit{
-					"l1": {
-						Rates: []kuadrantv1beta2.Rate{
-							{
-								Limit: 1, Duration: 3, Unit: kuadrantv1beta2.TimeUnit("minute"),
+				Defaults: kuadrantv1beta2.CommonSpec{
+					Limits: map[string]kuadrantv1beta2.Limit{
```

Comment:
Other places such as `limitador_cluster_envoyfilter_controller_test.go` still use implicit limits to test the implicit path still works as intended